xChrome.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.1.0.0'
+ModuleVersion = '1.0.2.0'
```

Comment:
Actually, please revert this change. Version should be increased, but only before it's ready to publish. It will be updated automatically by the publishing scripts